generators/chaincode/templates/typescript/src/contract-a.ts

```diff
@@ -15,24 +15,27 @@
  * limitations under the License.
  */
 
-'use strict';
+import { Contract } from 'fabric-contract-api';
 
-const { ChaincodeInterface, Shim } = require('fabric-shim');
+const assetType = 'WIDGETS';
 
-class Chaincode extends ChaincodeInterface {
+export class MyContract extends Contract  {
 
-    async Init(stub) {
-        const { fcn, params } = stub.getFunctionAndParameters();
-        console.info('Init()', fcn, params);
-        return Shim.success();
+    constructor(){
+        super('org.example.mycontract');
```

Comment:
Why is the MyContract contract in the mycontract namespace? Couldn't it just be `org.example`?